### PR TITLE
Cmake: Add a few more status lines during cmake

### PR DESCRIPTION
**Pull request by rgetz**
To better understand what is enabled/disabled - be a little more
specific; which will hopefully allow easier debugging for people.

Signed-off-by: Robin Getz <robin.getz@analog.com>

There are no functional changes to this change.

### CODE REVIEW

**Comment by rgetz** [comment]
yes - that is a better idea. 

Re-worked in this same branch.

File: bindings/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,17 @@
 if (CSHARP_BINDINGS)
 	add_subdirectory(csharp)
+else()
+	set(IIO_FEATURES_OFF "${IIO_FEATURES_OFF} C#-bindings")
```
Comment (pcercuei) [inline bindings/CMakeLists.txt:4]: Leftover from before?

Comment (rgetz) [reply bindings/CMakeLists.txt]: yes - should be fixed now.

**Review by pcercuei — APPROVED** [review]

**Comment by pcercuei** [comment]
This looks 🤩🤩🤩

**Comment by rgetz** [comment]
Just needs you to approve it :)

This is from the existing CI:

centos_7_x86_64
-- Features Enabled : xml zstd network dns-sd avahi ipv6 local usb utils iiod iiod-serial iiod-aio man-utils Python-bindings
-- Features Disabled: serial examples iiod-usb iiod-systemd iiod-sysvinit iiod-upstart doc man C#-bindings

centos_8_x86_64
-- Features Enabled : xml zstd network dns-sd avahi ipv6 local usb utils iiod iiod-serial iiod-aio iiod-usb man-utils Python-bindings
-- Features Disabled: serial examples iiod-systemd iiod-sysvinit iiod-upstart doc man C#-bindings

ubuntu_16_04_x86_64
-- Features Enabled : xml network dns-sd avahi ipv6 local usb utils examples iiod iiod-serial iiod-aio iiod-usb man man-utils Python-bindings
-- Features Disabled: zstd serial iiod-systemd iiod-sysvinit iiod-upstart doc C#-bindings

ubuntu_18_04_x86_64
-- Features Enabled : xml zstd network dns-sd avahi ipv6 serial local usb utils examples iiod iiod-serial iiod-aio iiod-usb man man-utils Python-bindings
-- Features Disabled: iiod-systemd iiod-sysvinit iiod-upstart doc C#-bindings

ubuntu_20_04_x86_64
-- Features Enabled : xml zstd network dns-sd avahi ipv6 serial local usb utils examples iiod iiod-serial iiod-aio iiod-usb man man-utils Python-bindings
-- Features Disabled: iiod-systemd iiod-sysvinit iiod-upstart doc C#-bindings

debian_buster_arm32v7
-- Features Enabled : xml zstd network dns-sd avahi ipv6 serial local usb utils examples iiod iiod-serial iiod-aio iiod-usb man man-utils Python-bindings
-- Features Disabled: iiod-systemd iiod-sysvinit iiod-upstart doc C#-bindings

debian_buster_arm64v8
-- Features Enabled : xml zstd network dns-sd avahi ipv6 serial local usb utils examples iiod iiod-serial iiod-aio iiod-usb man man-utils Python-bindings
-- Features Disabled: iiod-systemd iiod-sysvinit iiod-upstart doc C#-bindings

doxygen
-- Features Enabled : xml zstd network dns-sd avahi ipv6 serial local usb utils examples iiod iiod-serial iiod-aio iiod-usb doc man man-utils Python-bindings
-- Features Disabled: iiod-systemd iiod-sysvinit iiod-upstart C#-bindings

macOS_10_14
-- Features Enabled : xml zstd network dns-sd bonjour ipv6 usb utils examples man-utils Python-bindings
-- Features Disabled: serial doc man C#-bindings

macOS_10_15
-- Features Enabled : xml zstd network dns-sd bonjour ipv6 usb utils examples man-utils Python-bindings
-- Features Disabled: serial doc man C#-bindings

VS2019_Win32
-- Features Enabled : xml network dns-sd serial usb utils man-utils Python-bindings C#-bindings
-- Features Disabled: zstd ipv6 examples doc man

VS2019_Win64
-- Features Enabled : xml network dns-sd serial usb utils man-utils Python-bindings C#-bindings
-- Features Disabled: zstd ipv6 examples doc man

**Comment by pcercuei** [comment]
I usually wait a few hours to see if anyone else has remarks. I'll merge it later today if nobody complains.

**Comment by pcercuei** [comment]
Just tiny remarks:
- lowercase `enabled` and `disabled`,
- `python-bindings` and `csharp-bindings` (just because it looks weird if those two are the only ones that are not fully lowercase).

**Comment by rgetz** [comment]
No problem - nit pick away

**Comment by rgetz** [comment]
yes - that was on purpose.

**Pull request merged**